src/components/Common/DistributionBar/DistributionBarItem.tsx

```diff
@@ -1,25 +1,49 @@
 import React from 'react'
 
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+
+import { getFormattedPercentage } from '../../../helpers'
+
+import './DistributionBarItem.css'
 import DistributionBarPopup, { DistributionBarPopupContent } from './DistributionBarPopup'
-import HorizontalBar, { HorizontalBarProps } from './HorizontalBar'
 
 export type DistributionBarItemProps = {
   popupContent?: DistributionBarPopupContent
-} & HorizontalBarProps
+  value: number
+  className: string
+  selected?: boolean
+  onHover?: (e: React.MouseEvent<unknown>) => void
+  onBlur?: () => void
+}
+type Props = DistributionBarItemProps & { total: number }
 
-const DistributionBarItem = ({ value, className, selected, total, popupContent }: DistributionBarItemProps) => {
+const DistributionBarItem = ({ value, className, selected, total, popupContent, onHover, onBlur }: Props) => {
   if (value <= 0) {
     return null
   }
 
+  const horizontalBar = (
+    <div
+      className={TokenList.join([
+        'DistributionBarItem',
+        !!selected && 'DistributionBarItem--selected',
+        className,
+        !!selected && `${className}--selected`,
```

Comment:
I'm not 100% convinced that the prop selected also affects the custom className but I'm not sure, what do you think @andyesp ?

Reply:
Adding the className with the modifier (after the other className) will override the properties of the "original" class.